sklearnex/ensemble/tests/test_forest.py

```diff
@@ -63,8 +63,8 @@ def test_sklearnex_import_rf_classifier(dataframe, queue, block, trees, rows, sc
 
 @pytest.mark.parametrize("dataframe,queue", get_dataframes_and_queues())
 def test_sklearnex_import_rf_regression(dataframe, queue):
-    if queue and queue.sycl_device.is_gpu:
-        pytest.skip("RF regressor predict for the GPU sycl_queue is buggy.")
+    if (not daal_check_version((2025, "P", 200))) and queue and queue.sycl_device.is_gpu:
+        pytest.skip("Bug fixed in a later version.")

```

Comment:
id say what the bug was

Reply:
Do you mean to have the issue described in the skip message?

Reply:
sorry about the delay. you are exactly right

Reply:
Added a comment about the bug and version to the skip message.